helpers/content.go

```diff
@@ -359,6 +429,15 @@ func (c *RenderingContext) getConfig() *Blackfriday {
 	return c.Config
 }
 
+func (c *RenderingContext) getMmarkConfig() *Mmark {
+	c.configInit.Do(func() {
+		if c.MmarkConfig == nil {
+			c.MmarkConfig = NewMmark()
```

Comment:
This is will probably be correct in the end (I haven't looked closely), but just looking at this makes me ask "that doesn't look correct? Why don't you init all the configs?"

So what I would was:

Make a MmarkRenderingContext and BlackfridayRenderingContext and embed the common stuff from RenderingContext.